app/views/components/strong.scala.html

```diff
@@ -18,6 +18,6 @@
 
```

Comment:
Can this component be removed, thanks. Don't really see the point of it when only one view (EmailOrPostView) uses it as shown [here](https://github.com/search?q=repo%3Ahmrc%2Fcharities-registration-frontend+strong&type=code). A component should be reusable.